sdk/storage/azure-storage-blob-cryptography/src/main/java/com/azure/storage/blob/specialized/cryptography/EncryptedBlobClientBuilder.java

```diff
@@ -915,4 +920,15 @@ public EncryptedBlobClientBuilder requiresEncryption(boolean requiresEncryption)
         this.requiresEncryption = requiresEncryption;
         return this;
     }
+
+    /**
+     * Sets the encryption options for the blob.
+     *
+     * @param options The {@link BlobClientSideEncryptionOptions} for the blob.
+     * @return the updated EncryptedBlobClientBuilder object
+     */
+    public EncryptedBlobClientBuilder blobClientSideEncryptionOptions(BlobClientSideEncryptionOptions options) {
```

Comment:
Let's name the method `clientSideEncryptionOptions` to be consistent elsewhere across the classes.